app/scripts/angular-apimock.js

```diff
@@ -45,13 +45,49 @@ angular.module('apiMock', [])
 		var config = {
 			mockDataPath: '/mock_data',
 			apiPath: '/api',
-			disable: false
+			disable: false,
+			stripQueries: true
 		};
 		var fallbacks = [];
 
 		// Helper methods
 		//
 
+		function serialize(obj) {
+			var str = [];
+
+			obj = sortObjPropertiesAlpha(obj);
+			for(var p in obj){
+				if (obj.hasOwnProperty(p)) {
```

Comment:
Tip: In the rest of the code I've used [angular.forEach()](https://docs.angularjs.org/api/ng/function/angular.forEach) to have a [array.forEach()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) type of syntax, but still support the same browsers as Angular does. So we don't have to do the if-hasownproperty-dance if we don't want to. ^^